flake.nix

```diff
@@ -38,6 +39,19 @@
             "plutus-simple-model"
           ];
         })
+        ({
+          flake.config.herculesCI = {
+            onPush = {
+              mainChecks.outputs.mainCheck =
+                self.packages.x86_64-linux."hedgehog-plutus-simple:lib:hedgehog-plutus-simple".override
```

Comment:
i would move this to a flake output so it is possible to build with `nix build`, and reference it from there